scripts/compileAbiBin.js

```diff
@@ -39,12 +39,12 @@ fs.readdir("./build/contracts", function (err, files) {
         const path = './build/contracts/' + file
         let rawdata = fs.readFileSync(path);
         let contract = JSON.parse(rawdata);
-        let { abi, bytecode, deployedBytecode } = contract;
+        let { abi, bytecode} = contract;
         bytecode = bytecode.substring(2);
 
         if (abi.length === 0) return;
         fs.writeFileSync(ABI_PATH + basename + ".abi"  , JSON.stringify(abi));
         fs.writeFileSync(BIN_PATH + basename + ".bin", bytecode);
-        fs.writeFileSync(RUNTIME_PATH + basename + ".bin", deployedBytecode);
+        fs.writeFileSync(RUNTIME_PATH + basename + ".bin", bytecode);
```

Comment:
I think everything except the ABI stuff can be removed from here.

Reply:
Actually, both the ABI and bin are necessary

Reply:
So keep the BIN path but remove the Runtime path correct?